apps/champions/test/support/test_utils.ex

```diff
@@ -75,10 +75,10 @@ defmodule Champions.TestUtils do
       animation_trigger: 0,
       effects: [
         %{
-          type: "instant",
+          type: %{type: "instant"},
           initial_delay: 0,
           components: [],
-          modifier: [],
+          modifiers: [],
```

Comment:
If this isn't absolutely necessary, restore it so that it will generate less conflicts against #464